src/Bulk.php

```diff
@@ -296,6 +297,10 @@ public function send(): ResponseSet
      */
     protected function _processResponse(Response $response): ResponseSet
     {
+        switch ($response->getStatus()) {
```

Comment:
Would it be possible to cover this with a test?

Reply:
Yes, tests are added, pls check

Reply:
added